### PR TITLE
Mission completed! header was created

### PR DESCRIPTION
**Pull request by RatuhWD**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,57 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  font-family: "Roboto", sans-serif;
+}
```
Comment (AlisaVlasova) [inline src/style.css:5]:
```suggestion
html {
  font-family: "Roboto", sans-serif;
}
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,57 @@
...
+
+.header-container {
+  width: 1200px;
+}
+
+.header {
+  box-sizing: border-box;
```
Comment (AlisaVlasova) [inline src/style.css:12]: move box-sizing to * selector

File: src/style.css [hunk 3]
```diff
@@ -1 +1,57 @@
...
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.list__item:nth-last-child(-n+7) {
```
Comment (AlisaVlasova) [inline src/style.css:44]:
```suggestion
.list__item:not(:last-child) {
```

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Comment by RatuhWD** [comment]
- [DEMO LINK](https://ratuhwd.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ratuhwd.github.io/layout_moyo-header/report/html_report/)

File: src/style.css (hunk 1 above)
Comment (RatuhWD) [reply src/style.css]: ok

File: src/style.css (hunk 2 above)
Comment (RatuhWD) [reply src/style.css]: ok

File: src/style.css (hunk 3 above)
Comment (RatuhWD) [reply src/style.css]: ok

**Pull request closed without merging**